verify.py

```diff
@@ -96,7 +96,7 @@ def __init__(self, configfile, auth):
             elif opts[line] == '-d':
                 self.desc = opts[line + 1]
             elif opts[line] == '-b':
-                self.bin = opts[line + 1]
+                self.bin = self.desc
```

Comment:
I'm not sure that was ever tested on this version. It most likely would not have worked.  I'm increasingly leaning toward just changing all the existing config.bin/desc refs to config.dir, and updating the config parser accordingly, since it seems like this will be confusing going forward.